src/battle/action_cmd/air_lift.c

```diff
@@ -2,12 +2,64 @@
 
 #define NAMESPACE action_command_air_lift
 
-INCLUDE_ASM(s32, "battle/action_cmd/air_lift", func_802A9000_428A70);
+extern s32 D_80108AFC;
+extern s32 D_80108B28;
+extern s32 D_80292974;
+extern s32 D_80294340;
+extern s32 func_802A98B0_42EC50;
+
+ApiStatus func_802A9000_428A70(ScriptInstance* script, s32 isInitialCall) {
+    s32 *args;
+    HudElement* hudElement;
+
+    BattleStatus* battleStatus = &gBattleStatus;
+    ActionCommandStatus* actionCommandStatus = &gActionCommandStatus;
+
+    args = script->ptrReadPos;
+    battleStatus->unk_434 = &D_80294340;
+    battleStatus->unk_82 = 0;
+    battleStatus->unk_86 = 0x7F;
+    func_80268858();
+    actionCommandStatus->unk_5A = get_variable(script, *args++);
+    actionCommandStatus->actionCommandID = 0xD;
+    actionCommandStatus->unk_61 = 1;
+    actionCommandStatus->state = 0;
+    actionCommandStatus->unk_60 = 0;
+    actionCommandStatus->unk_44 = 0;
+    actionCommandStatus->unk_48 = 0;
+    actionCommandStatus->unk_68 = 0;
+    battleStatus->actionSuccess = 0;
+    battleStatus->unk_84 = 0;
+    actionCommandStatus->unk_6C = 0x1E;
+    actionCommandStatus->hudElementX = -0x30;
```

Comment:
This applies in other places in the PR as well, but we write coordinates as decimal numbers intead of hex